setup.cfg

```diff
@@ -4,7 +4,5 @@ dev = develop easy_install lasagne[testing]
 [pytest]
 addopts =
     -v --doctest-modules
-    --cov=lasagne --cov-report=term-missing
```

Comment:
Did you mean to commit this? We definitely don't want to get rid of coverage testing :)